January Newsletter (#659)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2023-01-25-newsletter.md

---
layout: post
title: January 2023 Newsletter
editor: Bob Turner
slug: 2023-01-25-newsletter
date: 2023-01-25T00:00:00Z+0100
tags:
category:
link:
description:
type: text
---
## The University of Sheffield Research Software Engineering Community Newsletter January 2023

Welcome to this month's newsletter for the research software community at The University of Sheffield, featuring news, opportunities, events and training for you.

### News, Web & Blogs

#### Web and Publications

* From the RSE Team blog [Who's to Blame?](https://rse.shef.ac.uk/blog/git-blame/) - Neil writes about using this unfortunately titled feature of `git` to better understand the history of code in a project.
* A new book with Sheffield-based co-authors "[Live Coding: A User's Manual](https://livecodingbook.toplap.org/)" Performative, improvised, on the fly: live coding is about how people interact with the world and each other via code. In the last few decades, live coding has emerged as a dynamic creative practice, gaining attention across cultural and technical fields—from music and the visual arts to computer science.

> * [Announcing the 2023 Software Sustainability Institute Fellows](https://software.ac.uk/blog/2022-12-15-announcing-2023-software-sustainability-institute-fellows) - Read this blog post from the Software Sustainability Institute.

Reviewer: ```suggestion
* [Announcing the 2023 Software Sustainability Institute Fellows](https://software.ac.uk/blog/2022-12-15-announcing-2023-software-sustainability-institute-fellows) - Read this blog post from the Software Sustainability Institute to find out more about the Fellowships they offer.
```

Author: They've awarded these already. I've clarified this.

> * A huge [Online Rust Course](https://google.github.io/comprehensive-rust/) has been made available by the Android team.

Reviewer: ```suggestion
* A comprehensive [Online Rust Course](https://google.github.io/comprehensive-rust/) has been made available by the Android team.
```

> * A really valuable perspective on [convincing academics to use git](https://www.youtube.com/watch?v=KgxKJjzh-Y4) (video) from Juulia Suvilehto at NormConf.

Reviewer: ```suggestion
* A really valuable perspective on [convincing academics to use git](https://www.youtube.com/watch?v=KgxKJjzh-Y4) (video) from Juulia Suvilehto at NormConf. More useful and interesting videos from [NormConf 2022](https://normconf.com/) are available see the [main track](https://www.youtube.com/watch?v=C9OlkY87vf8&list=PLYXaKIsOZBsu3h2SSKEovRn7rGy7wkUAV) and [lightning talks](https://www.youtube.com/watch?v=-6sS3wVYpM8&list=PLYXaKIsOZBstGUTXZXp2azDk8UJhpVVq3).
```

* [Love Data Week 2023: Exploring Dryad (webinar)](https://www.eventbrite.co.uk/e/love-data-week-2023-exploring-dryad-webinar-tickets-517737054467) - 17th Feb 2023 - ... explore how University of Sheffield researchers can get the most out of our membership of Dryad.
* [Introduction to Synthetic Data for Researchers](https://n8cir.org.uk/events/introduction-synthetic-data-researchers/) - 21st Feb 2023 - N8CIR event: This interactive workshop has two parts, the first of which introduces the basic concepts of synthetic data, shows examples, and explores why researchers need to be aware of what synthetic data is.
* [The Call for Proposals for DjangoCon Europe 2023 Is Now Open!](https://2023.djangocon.eu/news/announcing-call-for-proposals/) - closes 26th Febuary 2023
* [Python GPU Programming Workshop](https://n8cir.org.uk/events/python-gpu-programming-workshop/) - 18th Feb and 28th April 2023 - N8CIR event: Want to learn more about how to write code in Python that can take advantage of graphical processing units (GPUs)? Sign up for this two-part course introducing GPU programming through Python.

> * [ShinyConf 2023](https://shinyconf.appsilon.com/) - March 15-17, 2023, all-virtual

Reviewer: ```suggestion
* [ShinyConf 2023](https://shinyconf.appsilon.com/) - 15-17th March, 2023, all-virtual.
```

(Following @Robadob consistent date style :wink: )

> * [Intermediate Software Development Skills for Earth and Environmental Scientists:  Reproducible Research through Reusable, Reliable Code in Python](https://www.qualtrics.manchester.ac.uk/jfe/form/SV_87F94dIoT1aRwto) - 2023-03-27 to 2023-03-31, University of Manchester. Fancy improving your research techniques within the Earth, atmospheric and oceanic sciences? For any questions about the course and application, please contact Anja Le Blanc via e-mail at anja.leblanc@manchester.ac.uk.

Reviewer: ```suggestion
* [Intermediate Software Development Skills for Earth and Environmental Scientists:  Reproducible Research through Reusable, Reliable Code in Python](https://www.qualtrics.manchester.ac.uk/jfe/form/SV_87F94dIoT1aRwto) - 27th-31st March 2023, University of Manchester. Fancy improving your research techniques within the Earth, atmospheric and oceanic sciences? For any questions about the course and application, please contact Anja Le Blanc via e-mail at anja.leblanc@manchester.ac.uk.
```

### Training

#### Research IT Training

Research IT is providing a place for beginners or advanced users to expand their knowledge of HPC and different programming languages. The courses are part of the Doctoral Development Programme. For more information please visit our training registration web page (via VPN): [crs.shef.ac.uk](https://crs.shef.ac.uk).

#### RSE Team: Git & GitHub through GitKraken workshop - from Zero to Hero!

This is an introductory course, teaching the git and GitHub skills required to manage research code over it’s development lifecycle. These skills are essential for collaborative research teams.

* [Git & GitHub through GitKraken workshop 2023-03-06 & 2023-03-07](https://rse.shef.ac.uk/training/workshop/2023-03-06-git-zero-hero) 

> If a course is "sold out" please join the wait list - we regularly email people to encourage those that can no longer attend to cancel.

Reviewer: ```suggestion
If a course is "sold out" please join the wait list by signing up - we regularly email people to encourage those that can no longer attend to cancel. Those on the wait list get early notification when the courses are run again.
```

### Opportunities

#### Funding

> * [Artificial intelligence hubs for real data and for scientific and engineering research](https://www.ukri.org/opportunity/artificial-intelligence-hubs-for-real-data-and-for-scientific-and-engineering-research/) - Deadline 2023-02-09. Apply for funding for a research hub in either artificial intelligence (AI) for real data or AI for scientific and engineering research.

Reviewer: ```suggestion
* [Artificial intelligence hubs for real data and for scientific and engineering research](https://www.ukri.org/opportunity/artificial-intelligence-hubs-for-real-data-and-for-scientific-and-engineering-research/) - Deadline 9th February 2023. Apply for funding for a research hub in either artificial intelligence (AI) for real data or AI for scientific and engineering research.
```

> * [Artificial intelligence innovation to accelerate health research](https://www.ukri.org/opportunity/artificial-intelligence-innovation-to-accelerate-health-research/) - Closing date: 28 March 2023 - Apply for funding to develop innovative artificial intelligence (AI) technologies to be applied to health challenges.

Reviewer: ```suggestion
* [Artificial intelligence innovation to accelerate health research](https://www.ukri.org/opportunity/artificial-intelligence-innovation-to-accelerate-health-research/) - Closing date: 28th March 2023 - Apply for funding to develop innovative artificial intelligence (AI) technologies to be applied to health challenges.
```
